content/articles/localization-and-internationalization-in-android/index.md

```diff
@@ -0,0 +1,199 @@
+- `xml` - This holds arbitrary XML files that can be read at runtime, by calling the `resource.getXml()` method.
+
+To understand how each resource is localized, it's crucial to know the resource groups.
+
+How to access the resources:
+
+##### 1. In a `Class`  
+`[<res_directory>].<resource-type>.<Resource_name>`
+
+Examples:
+- `R.layout.activity-main`
+- `R.String.main`
+
+##### 2. XML file
+`@<resource-type>/<resource-name>`
+
+Examples:
+- `@String/word`
+- `@color/blue`
+
+### How the SDK handles localized resources
+Android selects localized resources to use based on the current device settings. The procedure is as follows:
+
+- The SDK eliminates qualifiers that contradict the device settings.
+- Then identifies the next qualifier in the order. If no resource subdirectory uses the qualifier, the subdirectory is also eliminated.
+- The process is repeated until the desired resource is left.
+
+### Localizing strings
+To localize strings appropriately, do not hard code them. Instead, use `string-resource` files to create strings value. 
+
+Follow the procedure below to localize a string:
+
+#### Step 1 - Creating a string file
+`Right-click` on resource `subdirectory values`, go to `new`, and select `resource files`.
+
+#### Step 2 - Naming string files
+Name the file similar to the available `string.xml` file on the wizard that appears. Then select `locale` on the available `qualifier` and navigate to the path below:
+
+![Create_string_file](/engineering-education/localization-and-internationalization-in-android/create_string_file.png)
+
+#### Step 3 - Choosing the language to localize your string
+Choose the language that you wish to localize on the same wizard. You can specify the region if required, then hit `OK`. An empty `string.xml` file will be generated.
+
+If you check the `string-resource` subdirectory, you will notice there are two `string.xml` files, with one having the specified language symbols.
+
+#### Step 4 - Writing to the newly created string file
+Finally, open your main `string.xml` file, copy the available strings, and paste them into the new `string.xml` file. 
+
+Once you have added the code, please do not change the `string name` to the original strings in the main file.
+
+You should instead change the string value to your desired language.
+
+### Localizing images 
+Its vital for an image to display the right content according to the context it is being utilized. For this reason, localization of images should be considered.
+
+This feature can be implemented as follows:
+
+#### Step 1 - Creating a new image subdirectory
+`Right-click` on the `resource` folder and navigate to the new `select android resource` directory.
+
+#### Step 2 - Naming newly created image subdirectory
+Name the directory similar to the available image directory and select drawable on the resource type on the wizard that appears under the available qualifier. 
```

Comment:
This section seems rushed. Highlight things such as folder names etc. Explain it more clearly.